src/slider/_slider.scss

```diff
@@ -16,382 +16,377 @@
+    $range-bg-focus-color 100%);
+    transform: scale(1);
+  }
+
+  &.is-lowest-value:active::-ms-thumb {
+    transform: scale(0.5625);
+    background: radial-gradient(circle closest-side,
+    transparent 0%,
+    transparent 77.78%,
+    $range-bg-color 77.78%,
+    $range-bg-color 100%);
+  }
+
+  &.is-lowest-value::-ms-fill-lower {
+    background: transparent;
+  }
+
+  &.is-lowest-value::-ms-fill-upper {
+    margin-left: 6px;
+  }
+
+  &.is-lowest-value:active::-ms-fill-upper {
+    margin-left: 9px;
+  }
+
+  /**************************** Disabled ****************************/
+
+  &:disabled:focus::-webkit-slider-thumb,
+  &:disabled:active::-webkit-slider-thumb,
+  &:disabled::-webkit-slider-thumb {
+    transform: scale(0.667);
+    background: $range-bg-color;
+  }
+
+  &:disabled:focus::-moz-range-thumb,
+  &:disabled:active::-moz-range-thumb,
+  &:disabled::-moz-range-thumb {
+    transform: scale(0.667);
     background: $range-bg-color;
-    flex: 0;
-    position: relative;
-    border: 0;
-    padding: 0;
-    transition: left 0.18s $animation-curve-default
   }
+
+  &:disabled +
+      .mdl-slider__background > .mdl-slider__background-lower {
+    background-color: $range-bg-color;
+    left: -6px;
+  }
+
+  &:disabled +
+      .mdl-slider__background > .mdl-slider__background-upper {
+    left: 6px;
+  }
+
+  &.is-lowest-value:disabled:focus::-webkit-slider-thumb,
+  &.is-lowest-value:disabled:active::-webkit-slider-thumb,
+  &.is-lowest-value:disabled::-webkit-slider-thumb {
+    border: 3px solid $range-bg-color;
+    background: transparent;
+    transform: scale(0.667);
+  }
+
+  &.is-lowest-value:disabled:focus::-moz-range-thumb,
+  &.is-lowest-value:disabled:active::-moz-range-thumb,
+  &.is-lowest-value:disabled::-moz-range-thumb {
+    border: 3px solid $range-bg-color;
+    background: transparent;
+    transform: scale(0.667);
+  }
+
+  &.is-lowest-value:disabled:active +
+      .mdl-slider__background > .mdl-slider__background-upper {
+    left: 6px;
+  }
+
+  &:disabled:focus::-ms-thumb,
+  &:disabled:active::-ms-thumb,
+  &:disabled::-ms-thumb {
+    transform: scale(0.25);
+    background: $range-bg-color;
+  }
+
+  &.is-lowest-value:disabled:focus::-ms-thumb,
+  &.is-lowest-value:disabled:active::-ms-thumb,
+  &.is-lowest-value:disabled::-ms-thumb {
+    transform: scale(0.25);
+    background: radial-gradient(circle closest-side,
+    transparent 0%,
+    transparent 50%,
+    $range-bg-color 50%,
+    $range-bg-color 100%);
+  }
+
+  &:disabled::-ms-fill-lower {
+    margin-right: 6px;
+    background: linear-gradient(to right,
+    transparent,
+    transparent 25px,
+    $range-bg-color 25px,
+    $range-bg-color 0);
+  }
+
+  &:disabled::-ms-fill-upper {
+    margin-left: 6px;
+  }
+
+  &.is-lowest-value:disabled:active::-ms-fill-upper {
+    margin-left: 6px;
+  }
+}
+
+// Some CSS magic to target only IE, by intentionally using bad syntax.
+.mdl-slider__input.mdl-slider__input\ {
```

Comment:
Ok, as long as it is getting removed later I'm good. I'd rather we used media-query targets for this kind of IE targeting (since it is possible) to keep the syntax hacks out.